### DISCUSSION TITLE
Unable to nest tables

### DISCUSSION THREAD

**Discussion by kollerp** [spectreconsole/spectre.console#725] [Q&A] [answered]
**Describe the bug**
I am unable to nest tables. 

**To Reproduce**
Table Table1 = new Table()
.AddColumn("column1")
.AddColumn("column2");

Table Table2 = new Table()
.AddColumn("column1")
.AddColumn("column2")
.AddRow("test", "test");

Table1.AddRow("test", table2);

This gives me a compile time error : 
'Table' does not contain a definition for 'AddRow' and the best extension method overload 'GridExtensions.AddRow(Grid, params string[])' requires a receiver of type 'Grid'

**Comment by patriksvensson** [comment, marked as answer]
@kollerp That method overload expects a params array of `string`; the one you're looking for takes a params array of `IRenderable`.

Try the following:

```csharp
Table1.AddRow(new Text("test"), table2);
```

**Reply by kollerp** [reply]
That did work, thanks!
